src/pages/api/auth/[...nextauth].ts

```diff
@@ -108,28 +108,36 @@ export const authOptions: NextAuthOptions = {
         intervalBetweenAttempts: 100
       });
 
-      const createCustomerWalletResponse = await holaplex.mutate<
-        CreateCustomerWalletData,
-        CreateCustomerWalletVars
-      >({
-        mutation: CreateCustomerWallet,
-        variables: {
-          input: {
-            customer: me.holaplexCustomerId,
-            assetType: process.env.HOLAPLEX_WALLET_ASSET_TYPE as AssetType
-          }
-        }
-      });
-
-      const wallet =
-        createCustomerWalletResponse.data?.createCustomerWallet.wallet;
-
-      await db.wallet.create({
-        data: {
-          holaplexCustomerId: me.holaplexCustomerId as string,
-          address: wallet?.address as string
-        }
-      });
+      const assetTypes = process.env.HOLAPLEX_WALLET_ASSET_TYPE?.split(
+        ','
+      ) as AssetType[];
+
+      await Promise.all(
+        assetTypes?.map(async (assetType) => {
+          const createCustomerWalletResponse = await holaplex.mutate<
+            CreateCustomerWalletData,
+            CreateCustomerWalletVars
+          >({
+            mutation: CreateCustomerWallet,
+            variables: {
+              input: {
+                customer: me.holaplexCustomerId,
+                assetType
+              }
+            }
+          });
+
+          const wallet =
+            createCustomerWalletResponse.data?.createCustomerWallet.wallet;
+
+          await db.wallet.create({
+            data: {
+              holaplexCustomerId: me.holaplexCustomerId as string,
+              address: wallet?.address as string
```

Comment:
Since we no have multiple wallet types can you drop the wallets table? We'll just add a resolve to the User object of the app that queries our API.